### PR TITLE
⬆️ Update dependencies related to security issues

### PR DESCRIPTION
**Pull request by Viicos**

### CODE REVIEW

File: requirements/dev.txt [hunk 1]
```diff
@@ -604,14 +604,15 @@ tornado==6.4
...
     #   -r requirements/ci.txt
+    #   black
     #   zgw-consumers
 uritemplate==4.1.1
     # via
     #   -r requirements/ci.txt
     #   coreapi
     #   drf-spectacular
     #   drf-yasg
-urllib3==1.26.15
```
Comment (Viicos) [inline requirements/dev.txt:614]: @annashamray do you remember why `urllib3` was pinned to `1.26.15` in https://github.com/maykinmedia/objecttypes-api/commit/170dd771d568aad07442f01bbc311793cbbf92a7? Is it still relevant?

Comment (annashamray) [reply requirements/dev.txt:614]: I think I had problems with OpenSSL and urllib3 2.0, and some github issue advised me to downgrade urllib3 and it fixed the problem. If you don't have import errors with openssl, then it should be fine

**Comment by annashamray** [comment]
I've created a fix for quickstart - https://github.com/maykinmedia/objecttypes-api/pull/110 , so please don't be scared by the failing quick start action

**Comment by annashamray** [comment]
Please rebase on master, it should fix CI

**Review by annashamray — APPROVED** [review]

**Pull request merged**